### PR TITLE
Change node-addon-api dependency in package.json file

### PR DESCRIPTION
**Pull request by anisha-rohra**

### CODE REVIEW

File: 1_hello_world/node-addon-api/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "private": true,
   "dependencies": {
     "bindings": "~1.2.1",
-    "node-addon-api": "^0.6.2"
```
Comment (mhdawson) [inline 1_hello_world/node-addon-api/package.json:9]: I think this needs to be 1.0.0 now right ?

File: 1_hello_world/node-addon-api/package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
...
+    "node-addon-api": "^0.6.3"
```
Comment (sampsongao) [inline 1_hello_world/node-addon-api/package.json]: Change to ^1.0.0

**Review by sampsongao — DISMISSED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by mhdawson** [comment]
Landed as a18255d9af0cabc18647715297479ce17699b52b

**Pull request closed without merging**